src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -621,12 +621,12 @@ public String getInvalidityInfoForTimeForVisibilityStartAndSessionStart(Date vis
      */
     public String getInvalidityInfoForTimeForVisibilityStartAndResultsPublish(Date visibilityStart,
                                                                               Date resultsPublish) {
-        return getInvalidtyInfoForFirstTimeIsBeforeSecondTime(visibilityStart, resultsPublish,
+        return getInvalidityInfoForFirstTimeIsBeforeSecondTime(visibilityStart, resultsPublish,
                 SESSION_VISIBLE_TIME_FIELD_NAME, RESULTS_VISIBLE_TIME_FIELD_NAME);
     }
 
-    private String getInvalidtyInfoForFirstTimeIsBeforeSecondTime(Date earlierTime, Date laterTime,
-            String earlierTimeFieldName, String laterTimeFieldName) {
+    private String getInvalidityInfoForFirstTimeIsBeforeSecondTime(Date earlierTime, Date laterTime,
+                                                                   String earlierTimeFieldName, String laterTimeFieldName) {
```

Comment:
I think that means @Bedol needs to relook at things, setting it back ongoing for now